site/assignments/tsp/tsp.rst

```diff
@@ -141,7 +141,7 @@ Assuming the GA is working and there are no serious issues, **5** points will be
 Enhancements and Modifications
 ------------------------------
 
-A total of **5** additional points will be awarded for enhancing and modifying the GA. In general, each
+A total of **5** additional points may be awarded for enhancing and modifying the GA. In general, each
```

Comment:
Specifying _up to X points_ makes it clearer as well, maybe. Though, again, dead horse, but, one more comma, a rubric would be clearer. There's these N sections worth f(x) points and you can only get up to M points.